compute/instances/troubleshooting/fix-common-issues.mdx

```diff
@@ -0,0 +1,296 @@
+- Implement log rotation:
+   - Configure `logrotate` or other log management solutions to prevent logs from filling the disk.
+
+### Additional resources
+- [Scaleway Documentation: How to attach a Block Storage volume to an Instance](/block-storage/how-to/attach-a-volume/)
+- [Scaleway Documentation: How to increase the size of your Block Storage volumes](/block-storage/how-to/increase-block-volume/)
+- [Linux Log Rotation Guide](https://linux.die.net/man/8/logrotate)
+
+## Packet loss issues with Instances
+
+### Symptoms
+- High latency or intermittent connectivity to your Instance.
+- Pings show dropped packets.
+- Applications that rely on constant connectivity (VoIP, real-time apps) experience disruptions.
+
+### Possible causes
+- Network congestion: On either the Instance side or your local network.
+- Incorrect MTU settings: The maximum transmission unit might cause fragmentation issues.
+- Firewall or rate limits: Overly strict firewall rules can drop packets.
+- Hardware or routing issues: Upstream or local network hardware problems.
+
+### Solution
+- Run basic network diagnostics:
+   - Use `ping` and `traceroute` from both your local machine and another external server.
+   - Check if packet loss is local or global.
+- Check the Instance's load:
+   - High CPU or network usage on your Instance can cause dropped packets. Monitor system resources with `top` or `htop`.
+- Optimize the MTU
+   - Try lowering the MTU (Maximum transmission unit) to 1400 or 1450 and see if packet loss improves, especially in VPN or container setups.
+- Adjust firewall/security groups:
+   - Confirm that your firewall is not inadvertently dropping packets or rate-limiting ICMP.
+
+### Additional resources
+- [Scaleway Documentation: How to use security groups](/instances/how-to/use-security-groups/)
+- [Scaleway Documentation: Preventing outgoing DDoS](/instances/reference-content/preventing-outgoing-ddos/)
+- [Scaleway Tutorial: Configuring Firewalls for Instances](/tutorials/installation-uncomplicated-firewall/)
+
+## I can't delete my Instance
+
+### Symptoms
+- Attempts to delete the Instance fail with an “Instance is protected” error.
+- You see a message in the Scaleway console that the Instance is protected or locked.
+
+### Possible causes
+- User-applied lock: You (or another administrator) enabled resource locking to prevent accidental deletion.
+
+### Solution
+- Disable the Protected Instance feature:
+   - In the Scaleway console, go to your Instance settings and turn off the lock if you set it manually.
+
+### Additional resources
+- [Scaleway Documentation: How to use the Protected Instance feature](/instances/how-to/use-protected-instance/)
+
+## Next steps
+
+- Always consult the official [Scaleway Instances Documentation](https://www.scaleway.com/en/docs/instances/) for detailed, up-to-date information to troubleshoot your Instance.
```

Comment:
Does the link work? I thought we have to only write it like this: [Link](/instances/)

Reply:
It does work but you are right!